requirements/edx/doc.in

```diff
@@ -5,3 +5,4 @@ code-annotations          # provides annotations for certain documentation
 sphinx-book-theme         # Common theme for all Open edX projects
 gitpython                 # fetch git repo information
 Sphinx                    # Documentation builder
+sphinx-design             # provides various responsive web-components
```

Comment:
```suggestion
sphinx-design             # provides various responsive web-components
sphinxcontrib-openapi[markdown]
```
Using base requirements, in order to get `make html` to pass, I also had to install `sphinxcontrib-openapi[markdown]`, so you might want to add that package here. 

Although, for what it's worth, that package does already exist in development.in. Not sure whether you see doc.in as parallel to development.in, or an extension of development.in.

Reply:
Added below with a comment.